server/pbench/bin/pbench-base.sh

```diff
@@ -124,12 +124,30 @@ function log_finish {
     exec 4>&-    # Close error file
 }
 
-# function used by the shims to quarantine problematic tarballs
+# Function used by the shims to quarantine problematic tarballs.  It
+# is assumed that the function is called within a log_init/log_finish
+# context.  Errors here are fatal but we log an error message to help
+# diagnose problems.
 function quarantine () {
+    set -x
```

Comment:
I guess this should be commented out to pass the unit tests.

Reply:
Yes - thanks! I'll squash everything down to the standard two commits later on today.